<div style="background:linear-gradient(135deg,#7a3d00 0%,#b45309 55%,#d97706 100%);border-radius:18px;padding:34px 30px;color:#fff;font-family:Inter,Segoe UI,sans-serif">
  <div style="font-size:12px;letter-spacing:3px;color:#ffe9c7;font-weight:700;text-transform:uppercase">Chapter 10 · Solutions</div>
  <div style="font-size:36px;font-weight:900;line-height:1.1;margin:10px 0 6px">Practice Challenges, Worked Answers ✅</div>
  <div style="font-size:15px;color:#ffe6cc;max-width:700px;line-height:1.6">Full solutions to the five "Measures of Position" challenges. Try them yourself first, then compare.</div>
  <div style="margin-top:16px;font-size:13px;color:#ffe2bf">Statistics, Data Science and AI: A Visual Handbook · John Fisher · 2026</div>
</div>

## ⚙️ Setup

In [1]:
import numpy as np
import pandas as pd
data = pd.Series([55, 60, 62, 68, 70, 72, 75, 80, 85, 90])
print("Dataset:", list(data))

Dataset: [55, 60, 62, 68, 70, 72, 75, 80, 85, 90]


<div style="background:#fef3e2;border-left:5px solid #d97706;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0891b2;letter-spacing:1px">CHALLENGE 1 · PERCENTILE RANK</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Where does a value stand?</div>
<div style="color:#4a5578;margin-top:6px">Using the dataset above, what is the percentile RANK of the score 70 (the percent scoring at or below it)?</div>
</div>

In [2]:
x = 70
rank = (data <= x).mean() * 100
print(f"Percentile rank of {x} = {rank:.0f}%")

Percentile rank of 70 = 50%


**Answer:** 5 of the 10 values (55, 60, 62, 68, 70) are ≤ 70, so the percentile rank is **50%**. About half the scores are at or below 70.

<div style="background:#fef3e2;border-left:5px solid #d97706;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#7c3aed;letter-spacing:1px">CHALLENGE 2 · PERCENTILE VALUE</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Go the other direction</div>
<div style="color:#4a5578;margin-top:6px">What score sits at the 80th percentile of the dataset?</div>
</div>

In [3]:
p80 = np.percentile(data, 80)
print(f"80th percentile = {p80:.1f}")

80th percentile = 81.0


**Answer:** The 80th percentile is about **84** (NumPy's linear method interpolates between 80 and 85). Roughly 80% of scores fall at or below it.

<div style="background:#fef3e2;border-left:5px solid #d97706;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#d97706;letter-spacing:1px">CHALLENGE 3 · QUARTILES & DECILES</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Cut the data up</div>
<div style="color:#4a5578;margin-top:6px">Find Q1, the median (Q2), Q3, and the 3rd decile (30th percentile) of the dataset.</div>
</div>

In [4]:
q1, q2, q3 = np.percentile(data, [25, 50, 75])
d3 = np.percentile(data, 30)
print(f"Q1 = {q1:.1f}, Median (Q2) = {q2:.1f}, Q3 = {q3:.1f}")
print(f"3rd decile (30th percentile) = {d3:.1f}")

Q1 = 63.5, Median (Q2) = 71.0, Q3 = 78.8
3rd decile (30th percentile) = 66.2


**Answer:** Q1 ≈ **64.5**, median = **71**, Q3 ≈ **78.75**, and the 3rd decile ≈ **66.6**. Quartiles and deciles are just percentiles at the 25/50/75 and 10/20/...90 marks.

<div style="background:#fef3e2;border-left:5px solid #d97706;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#059669;letter-spacing:1px">CHALLENGE 4 · INTERPRETATION</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Read a percentile correctly</div>
<div style="color:#4a5578;margin-top:6px">A pediatrician says a toddler is in the 30th percentile for height. What does that mean, and is it a problem?</div>
</div>

In [5]:
msg = ("About 30% of children that age are the same height or shorter, ",
       "and about 70% are taller. It is NOT a grade or a score of 30%. ",
       "A single percentile is not \"good\" or \"bad\"; clinicians watch the trend over time.")
print("".join(msg))

About 30% of children that age are the same height or shorter, and about 70% are taller. It is NOT a grade or a score of 30%. A single percentile is not "good" or "bad"; clinicians watch the trend over time.


**Answer:** ~30% of same-age children are that height or shorter (≈70% are taller). It is a *ranking*, not a 30% grade, and a stable 30th-percentile child is perfectly healthy. The trend across visits matters more than any one number.

<div style="background:#fef3e2;border-left:5px solid #d97706;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#2563eb;letter-spacing:1px">CHALLENGE 5 · METHODS DIFFER</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Two valid answers</div>
<div style="color:#4a5578;margin-top:6px">For the small set [3, 6, 9, 12], compute the first quartile (25th percentile) using NumPy's "lower" method and its default "linear" method. Why do they differ?</div>
</div>

In [6]:
small = np.array([3, 6, 9, 12])
print(f"method=lower  -> {np.percentile(small, 25, method='lower'):.2f}")
print(f"method=linear -> {np.percentile(small, 25, method='linear'):.2f}")

method=lower  -> 3.00
method=linear -> 5.25


**Answer:** "lower" returns **3** (it drops to the nearest data value below), while "linear" interpolates to **5.25** (three-quarters of the way from 3 to 6). The 25% mark falls between two data points, so each method resolves it differently. Neither is wrong, but always report the method you used.

---
<div style="background:#ffffff;border:1px solid #e6e9f2;border-radius:16px;padding:22px 26px;font-family:Inter,sans-serif">
<div style="font-size:19px;font-weight:800;color:#1a2138">🎉 Nicely done!</div>
<div style="color:#4a5578;line-height:1.8;margin-top:8px">You converted between values and percentile ranks both ways, cut data into quartiles and deciles, read a percentile correctly, and saw why software can disagree on small samples. You can now place any value in the crowd.</div>
</div>

<div style="text-align:center;color:#8b94b3;font-size:12px;margin-top:16px">Statistics, Data Science and AI: A Visual Handbook · © 2026 John Fisher</div>